clients/gtest/hipblas_gtest_main.cpp

```diff
@@ -78,15 +78,6 @@ static void print_version_info()
     // clang-format on
 }
 
-int hipblas_test_datafile()
-{
-    int ret = 0;
-    for(Arguments arg : HipBLAS_TestData())
-        ret |= run_bench_test(arg, 1, 0);
-    test_cleanup::cleanup();
-    return ret;
-}
```

Comment:
We do have new requests to run reduced tests, so maybe we should implement yaml file input like rocblas?  How much time before tests run?  Anyway see your comment at end of file:
// Temporarily running all tests while transitioning to YAML based testing for hipblas-test suite.
2 years ago :-)